generate-grpc-stubs.sh

```diff
@@ -20,8 +20,8 @@ echo "#######################################################"
 set -x
 
 ROOT_DIR=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )" )
+python3 -m grpc_tools.protoc -I ./velocitas_sdk/proto --grpc_python_out=./velocitas_sdk/proto --python_out=./velocitas_sdk/proto --mypy_out=./velocitas_sdk/proto ./velocitas_sdk/proto/**/*.proto
```

Comment:
Why remove the empty line? Looks weird in diff ;-)

Reply:
reverted